frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -74,7 +74,7 @@ type OwnProps = {
 type PublicOrEmbeddedDashboardProps = OwnProps &
   ReduxProps &
   DashboardDisplayOptionControls &
-  EmbedDisplayParams &
+  Omit<EmbedDisplayParams, "font"> &
```

Comment:
The `font` field was not used inside this component (because it was set to the store directly in `use-embed-font` hook.

I removed it from types and from being passed to this component as a prop

Reply:
I reverted this change as well and will extract it to a separate PR